src/pybind/mgr/progress/module.py

```diff
@@ -607,53 +620,59 @@ def _pg_state_changed(self, pg_dump):
         except ZeroDivisionError:
             return
         if progress < 1.0:
+            self.log.warning("Starting Global Recovery Event,\
+              {0} pgs not in active + clean state \
+              ".format(total_pg_num - active_clean_num))
```

Comment:
nit, i'd suggest to be more consistent when printing logging message, we have
```py
log.debug("Updated progress to %s", self.summary())
```
couple lines above. so, would be better to put
```py
self.log.warning(("Starting Global Recovery Event,"
                  "%d pgs not in active + clean state"),
                 total_pg_num - active_clean_num)
```